app/views/posts/_post.html.erb

```diff
@@ -36,15 +36,14 @@
         </div>
         <div class="timeline-likes">
             <div class="stats-right">
-                <span class="stats-text">259 Shares</span>
-                <span class="stats-text">21 Comments</span>
+                <span class="stats-text"><%= post.comments.count %> Comments</span>
             </div>
             <div class="stats">
                 <span class="fa-stack fa-fw stats-icon">
                     <i class="fa fa-circle fa-stack-2x text-primary"></i>
                     <i class="fa fa-thumbs-up fa-stack-1x fa-inverse"></i>
                 </span>
-                <span class="stats-total">4.3k</span>
+                <span class="stats-total"><%= post.likes.count %></span>
```

Comment:
This will perform a db query in the view file, try to send this information from controller itself. You can join likes with posts in the controller

Reply:
will do that! thanks